widgetastic_manageiq.py

```diff
@@ -1205,15 +1206,15 @@ class ItemsToolBarViewSelector(View):
         view_selector.selected
     """
     ROOT = './/div[contains(@class, "toolbar-pf-view-selector")]'
-    grid_button = Button(title='Grid View')
```

Comment:
I wasn't seeing these on the infra providers, but you're right the containers provider details page has them so I'll have to implement there as well.